app/src/main/java/nerd/tuxmobil/fahrplan/congress/about/AboutDialog.kt

```diff
@@ -0,0 +1,183 @@
+package nerd.tuxmobil.fahrplan.congress.about
+
+import nerd.tuxmobil.fahrplan.congress.extensions.requireViewByIdCompat
+import nerd.tuxmobil.fahrplan.congress.utils.LinkMovementMethodCompat.getInstance
+import nerd.tuxmobil.fahrplan.congress.extensions.toSpanned
+import nerd.tuxmobil.fahrplan.congress.extensions.setLinkText
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.os.Bundle
+import nerd.tuxmobil.fahrplan.congress.R
+import androidx.core.content.ContextCompat
+import android.view.View
+import android.widget.TextView
+import androidx.annotation.MainThread
+import androidx.fragment.app.DialogFragment
+import nerd.tuxmobil.fahrplan.congress.BuildConfig
+
+class AboutDialog : DialogFragment() {
+
+    companion object {
+        const val FRAGMENT_TAG = "about"
+        private const val BUNDLE_KEY_SCHEDULE_VERSION = BuildConfig.APPLICATION_ID + ".BUNDLE_KEY_SCHEDULE_VERSION"
+        private const val BUNDLE_KEY_SUBTITLE = BuildConfig.APPLICATION_ID + ".BUNDLE_KEY_SUBTITLE"
+        private const val BUNDLE_KEY_TITLE = BuildConfig.APPLICATION_ID + ".BUNDLE_KEY_TITLE"
+
+        fun newInstance(
+            scheduleVersion: String,
+            subtitle: String,
+            title: String
+        ): AboutDialog {
+            val args = Bundle()
+            args.putString(BUNDLE_KEY_SCHEDULE_VERSION, scheduleVersion)
+            args.putString(BUNDLE_KEY_SUBTITLE, subtitle)
+            args.putString(BUNDLE_KEY_TITLE, title)
+
+            val dialog = AboutDialog()
+            dialog.arguments = args
+
+            return dialog
+        }
+    }
+
+    private var scheduleVersionText = ""
+    private var subtitleText = ""
+    private var titleText = ""
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View = inflater.inflate(R.layout.about_dialog, container, false)
+
+    @MainThread
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val arguments = requireArguments()
+
+        setStyle(STYLE_NO_TITLE, R.style.Dialog)
+
+        scheduleVersionText = arguments.getString(BUNDLE_KEY_SCHEDULE_VERSION, "")
+        subtitleText = arguments.getString(BUNDLE_KEY_SUBTITLE, "")
+        titleText = arguments.getString(BUNDLE_KEY_TITLE, "")
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        // Schedule Version
+        val scheduleVersion = view.requireViewByIdCompat<TextView>(R.id.about_session_version_view)
+        if (scheduleVersionText.isEmpty()) {
+            scheduleVersion.visibility = View.GONE
```

Comment:
Use `isVisible` here as in `scheduleVersion.isVisible = false` - and everywhere else.